flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
WHY?

Reply:
Because I would have to build and test the application on `aarch64` to follow submission requirements. Although I don't have any (virtual or not) machine using this architecture now.

If `aarch64` support is an hard requirement to submit application on Flathub I'll take time to try build and run (crossing fingers to not need to update too many parts of the game).

Reply:
so no specific reason. It's clear that not building it doesn't allow anyone to test it.